awx/ui/client/lib/components/launchTemplateButton/launchTemplateButton.component.js

```diff
@@ -19,6 +21,11 @@ function atLaunchTemplateCtrl (
     const jobTemplate = new JobTemplate();
     const workflowTemplate = new WorkflowTemplate();
     vm.strings = componentsStrings;
+    vm.strings.get('launchTemplate.DEFAULT');
```

Comment:
Should this be `vm.launchTooltip =  vm.strings.get('launchTemplate.DEFAULT');` ?